pwiz_tools/Skyline/Model/SrmDocument.cs

```diff
@@ -1482,11 +1482,13 @@ private SrmDocument MergeMatchingPeptidesUserInfo(IList<PeptideGroupDocNode> pep
             return docNew;
         }
 
-        public MassListImporter PreImportMassList(MassListInputs inputs, IProgressMonitor progressMonitor, bool tolerateErrors)
+        public MassListImporter PreImportMassList(MassListInputs inputs, IProgressMonitor progressMonitor, bool tolerateErrors, DOCUMENT_TYPE inputType = DOCUMENT_TYPE.none)

```

Comment:
a little explanatory comment header here would be nice

Reply:
I added a comment describing the method and the parameter I added. Is this the comment format you are looking for? I am not sure what the best comment style is when I only want to comment on one of the parameters.

Reply:
If you don't want to go the full Monty on the comment, then just put it on its own line and comment that:
       /// <summary>
        /// Return a mass list import if the progress monitor is not cancelled and we are able to read the document
        /// </summary>
       public MassListImporter PreImportMassList(MassListInputs inputs, IProgressMonitor progressMonitor, bool tolerateErrors,
            DOCUMENT_TYPE inputType = DOCUMENT_TYPE.none) // "None" means "don't know if it's peptides or small molecules, go figure it out".